loopdb/sqlite.go

```diff
@@ -309,14 +308,30 @@ func (r *SqliteTxOptions) ReadOnly() bool {
 	return r.readOnly
 }
 
+// parseTimeStamp tries to parse a timestamp string with both the
+// parseSqliteTimeStamp and parsePostgresTimeStamp functions.
+// If both fail, it returns an error.
+func parseTimeStamp(dateTimeStr string) (time.Time, error) {
+	t, err := parseSqliteTimeStamp(dateTimeStr)
+	if err != nil {
+		t, err = parsePostgresTimeStamp(dateTimeStr)
+		if err != nil {
+			return time.Time{}, fmt.Errorf("invalid timestamp format: %v",
```

Comment:
nit: i think we can jus return the error here since it's already the same string.